cypress/support/commands.ts

```diff
@@ -0,0 +1,49 @@
+export enum AuthRoles {
+  ChildrensGroup = 'ChildrensGroup',
+  AdultsGroup = 'AdultsGroup',
+  AdminDevGroup = 'AdminDevGroup',
+}
+
+type RoleConfig = {
+  tokenValue: string;
+};
+
+const roleConfigurations: Record<AuthRoles, RoleConfig> = {
+  ChildrensGroup: {
+    tokenValue: Cypress.env('TEST_KEY_CHILDREN_GROUP'),
+  },
+  AdultsGroup: {
+    tokenValue: Cypress.env('TEST_KEY_ADULT_GROUP'),
+  },
+  AdminDevGroup: {
+    tokenValue: Cypress.env('TEST_KEY_ADMIN_DEV'),
+  },
+};
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Cypress {
+    interface Chainable {
+      visitAs: typeof visitAs;
+    }
+  }
+}
+
+const visitAs = (
```

Comment:
Ah, so this a custom method we're adding to Cypress?

Reply:
That's right, I've effectively extended the standard `cy.visit` with new functionality that makes more sense to us (and makes our tests a little cleaner).